### PR TITLE
initiate project

### PR DESCRIPTION
**Pull request by mirrou**

### CODE REVIEW

**Comment by reggieroby** [comment]
its not only my review. you all should add your review as well.

File: travel_hive/src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1 className="App-title">TRAVEL HIVE</h1>
```
Comment (reggieroby) [inline travel_hive/src/App.js]: awesome

**Comment by mirrou** [comment]
woot!  Thanks team :)

**Comment by myburt** [comment]
Everything looks great. Sorry for the late pickup on this I was trying to poke around with react. but looks good.

**Pull request merged**